### ISSUE TITLE
Running on Windows 

### ISSUE THREAD

**Issue by abhishekpuro** [waldekmastykarz/docker-spfx#13]
Hi, 

When I use the below code to get the docker image. I get below error on windows platform 
docker pull waldekm/spfx:online 

image operating system "linux" cannot be used on this platform 

Even if I use the command docker run. I get the same error 
docker run -it --rm --name spfx-helloworld -v /c/TTBL/Demo/DemoSPFx:/usr/app/spfx -p 5432:5432 -p 4321:4321 -p 35729:35729 waldekm/spfx:online

**Comment by waldekmastykarz** [comment]
Do you have Hyper-V enabled on your machine?

**Comment by waldekmastykarz** [comment]
Closing due to lack of response.

**Comment by AmolM27** [comment]
@waldekmastykarz I get the same problem. I am on a Oracle Virtual Box which I got from https://developer.microsoft.com/en-us/windows/downloads/virtual-machines. I am on Windows Containers in Docker and get the error as reported.

If I tried to shift to Linux Containers, I get error "Unable to start: The running command stopped because the preference variable "ErrorActionsPreference" or common parameter is set to Stop: 'MobyLinuxVM' failed to start. Failed to start the virtual machine 'MobyLinuxVM' because of the Hyper-V components is not running "

I had 'Hyper-V' features enabled, I turned it off and rebooted and get the same issues. Any suggestions?
